SECURITY.md

```diff
@@ -62,7 +62,7 @@ In Maven, you can use the [os-maven-plugin](https://github.com/trustin/os-maven-
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>1.1.33.Fork26</version>
+      <version>2.0.3.Final</version>
```

Comment:
Then make a new commit and apply it to both master and v1.4.x. It is really important not to change cherrypicks when unnecessary and it seems unrelated to this PR.